Sources/Queues/QueueWorker.swift

```diff
@@ -56,11 +58,12 @@ public struct QueueWorker {
             return complete
         }.flatMapError { error in
             if remainingTries == 0 {
-                logger.error("Job failed: \(error)")
+                logger.error("Job \(name) failed with error: \(error)")
                 return job._error(self.queue.context, error, payload: payload)
             } else {
-                logger.error("Retrying job: \(error)")
+                logger.error("Job \(name) failed, retrying: \(error)")
                 return self.run(
+                    name: name,
```

Comment:
For this section if we're going to add in the job name to the log I think we should also add the job's ID. And also add those into the metadata instead of the body of the text